src/components/10-atoms/toggle-switch/index.scss

```diff
@@ -0,0 +1,60 @@
+$toggle-switch-animation: 0.4s ease;
+$toggle-switch-height: 30px;
+$toggle-switch-width: 50px;
+$toggle-switch-spacing: 2px;
+
+.a-toggle-switch {
+  display: block;
+  position: relative;
+  height: $toggle-switch-height;
+  width: $toggle-switch-width;
+
+  &__input {
+    display: none;
+
+    &:checked {
+      & + .a-toggle-switch__slider {
+        background-color: $color-malachite;
+      }
+
+      & + .a-toggle-switch__slider::before {
+        $distance: $toggle-switch-width - $toggle-switch-height;
+        transform: translateX($distance);
```

Comment:
Move `$distance` to the other variables on top of file. Or remove that variable and calculate directly: `translateX(w - h)`